### ISSUE TITLE
`no-invisible-character`: false positive on space in `RegExp` constructor

### ISSUE THREAD

**Issue by zloirock** [ota-meshi/eslint-plugin-regexp#243]
```js
RegExp(' ') // => Unexpected invisible character. Use ' ' instead  regexp/no-invisible-character
```
